src/config/Config.zig

```diff
@@ -1350,9 +1350,18 @@ keybind: Keybinds = .{},
 /// Specified as either hex (`#RRGGBB` or `RRGGBB`) or a named X11 color.
 @"window-titlebar-foreground": ?Color = null,
 
-/// current gtk implementation for color management is not good enough.
-/// see: https://bugs.kde.org/show_bug.cgi?id=495647
-@"disable-gtk-color-mgmt": bool = false,
+/// Controls whether to disable GDK color management in GTK applications.
+///
+/// By default this is set to `false`, meaning color management is enabled.
+/// You may want to enable this setting (set to `true`) if you experience:
+/// - Incorrect or washed out colors in your terminal
+/// - Color inconsistencies between GTK applications
+/// - Performance issues related to color management
+///
+/// This is a workaround for known issues with GTK's color management implementation,
+/// particularly affecting applications running under Wayland.
+/// See: https://bugs.kde.org/show_bug.cgi?id=495647
```

Comment:
This looks good, but is there a corresponding GTK bug that we can link to?

Reply:
After a *lot* of digging it appears that this is the corresponding GTK MR: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/7612, fixed in [4.15.6](https://gitlab.gnome.org/GNOME/gtk/-/blob/15edfe4ffa7a800ab67717e6a6be81d0b37c1813/NEWS#L28)